yorkie/src/main/kotlin/dev/yorkie/document/crdt/RHTPQMap.kt

```diff
@@ -0,0 +1,158 @@
+package dev.yorkie.document.crdt
+
+import dev.yorkie.document.time.TimeTicket
+import dev.yorkie.util.MaxPriorityQueue
+import dev.yorkie.util.PQNode
+import dev.yorkie.util.YorkieLogger
+
+/**
+ * RHTPQMap is replicated hash table with priority queue by creation time.
```

Comment:
According to the acronym rule, I think we should rename `RHTPQMap` to `RhtPQMap`:
```suggestion
 * [RhtPQMap] is replicated hash table with a priority queue by creation time.
```
https://kotlinlang.org/docs/coding-conventions.html#choose-good-names